### PR TITLE
Update all third-party component versions - Merge to Dev

### PR DESCRIPTION
**Pull request by sunithasomanna**
Merge the following updates to Dev branch: 
- Updated the Getting Started Readme with the latest versions for K8S and its components, few third-party components, Docker, Docker images of K8S and ODIM services and their respective sample output screenshots.
- Also, renamed the following:
   - kube-master - kube_control_plane
   - kube-node - kube_node

### CODE REVIEW

**Review by sukhwinder20 — CHANGES_REQUESTED** [review]
1. Download and install Ubuntu 18.04 LTS--> change to ubuntu-20.04.3-live-server-amd64.iso

2. sudo apt-get install python3.8=3.8.0-3~18.04.1 -y->not found> change it to "sudo apt-get install python3.8=3.8.10-0ubuntu1~20.04.2 -y"

3.	sudo apt-get install python3-pip=9.0.1-2.3~ubuntu1.18.04.5 -y--> not found>>change it to "sudo apt-get install python3-pip=20.0.2-5ubuntu1.6 -y"

4.sudo apt-get install software-properties-common=0.96.24.32.14 -y-->not found>>change it to "sudo apt-get install software-properties-common=0.99.9.8 -y"

5.sudo apt-get install openjdk-11-jre-headless=11.0.11+9-0ubuntu2~18.04 -y-->not found>>change it to "sudo apt-get install openjdk-11-jre-headless=11.0.13+8-0ubuntu1~20.04 -y"

6. sudo apt-get install -y apt-transport-https=1.6.12ubuntu0.2 ca-certificates=20210119~18.04.1 curl=7.58.0-2ubuntu3.14>>not found>>change it to "sudo apt-get install -y apt-transport-https=2.0.6  ca-certificates=20210119~20.04.2 curl=7.68.0-1ubuntu2.7"

7.sudo apt-get install -y gnupg-agent=2.2.4-1ubuntu1.3 software-properties-common=0.96.24.32.14>> not found >> change it to "sudo apt-get install -y gnupg-agent=2.2.19-3ubuntu2.1 software-properties-common=0.99.9.8"

8. sudo apt-get install -y docker-ce=5:19.03.12~3-0~ubuntu-bionic docker-ce-cli=5:19.03.12~3-0~ubuntu-bionic containerd.io --allow-downgrades not found >> change it to "sudo apt-get install -y docker-ce=5:20.10.12~3-0~ubuntu-focal docker-ce-cli=5:20.10.12~3-0~ubuntu-focal  containerd.io --allow-downgrades"

File: README.md [hunk 1]
```diff
@@ -451,19 +466,19 @@ The following table lists the software components and their versions that are co
 
     | **Docker image name** | **Version** | **Docker image bundle name** |
     | :-------------------- | ----------- | ---------------------------- |
-    | account-session       | 1.0         | account-session.tar          |
```
Comment (sukhwinder20) [inline README.md:454]: Add lenovoplugin 1,0

File: README.md [hunk 2]
```diff
@@ -451,19 +466,19 @@ The following table lists the software components and their versions that are co
```
Comment (sukhwinder20) [inline README.md:466]: change api:1.0 to api:2.0

File: README.md (hunk 1 above)
Comment (sunithasomanna) [reply README.md:454]: added

File: README.md (hunk 2 above)
Comment (sunithasomanna) [reply README.md:466]: changed

**Comment by sunithasomanna** [comment]
Made changes in the sub-branch and raising a new PR for merging to dev. Closing this PR

**Comment by sunithasomanna** [comment]
Team has more feedback in this section. Hence closing this PR and raising a new PR with latest sprint number.

**Pull request closed without merging**